<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableView</class>
 <widget class="QWidget" name="TableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayoutWidgets"/>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="toolButtonUp">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBoxDisable">
       <property name="text">
        <string>Disable all items</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="toolButtonClear">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="11">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="8">
    <widget class="QToolButton" name="toolButtonExport">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QToolButton" name="toolButtonImport">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QToolButton" name="toolButtonDelete">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QToolButton" name="toolButtonAdd">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="toolButtonDown">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
